packages/flutter_tools/lib/src/android/gradle_errors.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
This is failing CI because we ban the import of dart:io:

```
05:14 +30 ~10 -1: test/integration.shard/forbidden_imports_test.dart: no unauthorized imports of dart:io [E]                                                                                           
  lib/src/android/gradle_errors.dart imports 'dart:io'; import 'lib/src/base/io.dart' instead
  package:matcher                                           fail
  test/integration.shard/forbidden_imports_test.dart 84:13  main.<fn>
  test/src/common.dart 184:18                               test.<fn>
```

https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8757429048283166209/+/u/run_test.dart_for_tool_integration_tests_shard_and_subshard_3_4/test_stdout